deltachat-ios/AppDelegate.swift

```diff
@@ -23,6 +23,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
     var window: UIWindow?
     var notifyToken: String?
     var applicationInForeground: Bool = false
+    private var launchOptions: [UIApplication.LaunchOptionsKey: Any]?
+    private var pendingAppInitialization = true;
```

Comment:
```suggestion
    private var pendingAppInitialization = true
```

semicolon not needed.

i would also prefer a more concrete naming for this boolean, sth. as `appFullyInitialized` that is set to `false` here and set to `true` at the end of `finishAppInitialization`